### PR TITLE
BF-149 Filter by bundle

### PR DESCRIPTION
**Pull request by var-kyle**
# Description

[BF-149 Filter by bundle](https://bf-decd.atlassian.net/browse/BF-149?atlOrigin=eyJpIjoiMTBmYjk0MGUwNDExNDBmMzhlZjg4NTA5ZmI4YmQ5OGEiLCJwIjoiaiJ9)

Add the ability to filter by selected bundles. When clicking on a bundle, the id of the bundle is added to the state and the list of benefits is then filtered to only show benefits that are a part of the selected bundle.

## Meets Definition of Done
- [x] Unit tests up to date
- [ ] E2E tests up to date
- [x] I18n

### CODE REVIEW

**Review by shewood — APPROVED** [review]
Looks great. I added a bundle and it appeared as expected. I am just not sure if they are supposed to be selectable as the popular categories was?

**Comment by var-kyle** [comment]
I found the problem with the box being selectable and pushed a fix, please take a look again to see if it's alright.

File: pages/index.js [hunk 1]
```diff
@@ -89,8 +93,36 @@ export default function Home({ locale, lifeBundles, situationCookie }) {
...
+                // if it hasn't, add it to the list
+                filteredData.push({ benefit: benefit });
+              }
+            }
+          }
+        }
+      }
+    }
+    setBenefits(filteredData.length > 0 ? filteredData : data);
+  }, [situation, bundles]);
```
Comment (DaceyTom2) [inline pages/index.js:125]: This logic is hard to follow

**Review by DaceyTom2 — APPROVED** [review]
Good work

**Review by shewood — COMMENTED** [review]
Looks good. The highlight now shows when selected.
